server/index.js

```diff
@@ -124,7 +124,7 @@ app.post('/auth/register', (req, res) => {
       User.create({
         username,
         password,
-        coinCount: 14,
+        coinCount: 400,
```

Comment:
this is fine for development but before we present on monday, we should probably reset to 0 so new users don't start off with coins